JSONSchema/schema.json

```diff
@@ -0,0 +1,168 @@
+                                }
+                            }
+                        }
+                    },
+                    "additionalProperties":false
+                },
+                "transaction":{
+                    "type":"object",
+                    "properties":{
+                        "data":{
+                            "type":"array",
+                            "items":{
+                                "type":"string",
+                                "pattern":"^(0x[0-9a-f]*)?$"
+                            }
+                        },
+                        "gasLimit":{
+                            "type":"array",
+                            "items":{
+                                "type":"string",
+                                "pattern":"^0x[0-9a-f]*$"
+                            }
+                        },
+                        "gasPrice":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
+                        },
+                        "nonce":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
+                        },
+                        "secretKey":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
+                        },
+                        "to":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
+                        },
+                        "value":{
+                            "type":"array",
+                            "items":{
+                                "type":"string",
+                                "pattern":"^0x[0-9a-f]*$"
+                            }
+                        }
+                    },
+                    "additionalProperties":false
+                }
+            }
+        }
+    },
+    "definitions": {
+        "TransactionResults": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "hash": {
+                        "type": "string",
+                        "pattern":"^0x[0-9a-f]*$"
+                    },
+                    "indexes": {
+                        "type": "object",
+                        "properties": {
+                            "data": {
+                                "type": "integer"
+                            },
+                            "gas": {
+                                "type": "integer"
+                            },
+                            "value": {
+                                "type": "integer"
+                            }
+                        },
+                        "additionalProperties": false
+                    }
+                },
+                "additionalProperties": false
+            }
+        },
+        "HexString": {
```

Comment:
That's an oversight on my part.  It's being deleted and replaced with several different sub-schema definitions.